### ISSUE TITLE
Error while turn off command

### ISSUE THREAD

**Issue by wacki4** [custom-components/media_player.braviatv_psk#39]
**Describe the issue**
On running on automation - tv don't turn off and there is an error in homeassistant log.

**Expected behavior**
TV is turned off, but home assistant try to turn it off with this error.

**Home Assistant version**
0.110.4

**Output of HA logs**
```
Traceback (most recent call last):
  File "/srv/homeassistant/lib/python3.7/site-packages/homeassistant/helpers/script.py", line 153, in _async_step
    self, f"_async_{cv.determine_script_action(self._action)}_step"
  File "/srv/homeassistant/lib/python3.7/site-packages/homeassistant/helpers/script.py", line 623, in _async_call_service_step
    *self._prep_call_service_step(), blocking=True, context=self._context
  File "/srv/homeassistant/lib/python3.7/site-packages/homeassistant/core.py", line 1253, in async_call
    task.result()
  File "/srv/homeassistant/lib/python3.7/site-packages/homeassistant/core.py", line 1288, in _execute_service
    await handler.func(service_call)
  File "/srv/homeassistant/lib/python3.7/site-packages/homeassistant/helpers/entity_component.py", line 213, in handle_service
    self._platforms.values(), func, call, required_features
  File "/srv/homeassistant/lib/python3.7/site-packages/homeassistant/helpers/service.py", line 454, in entity_service_call
    future.result()  # pop exception if have
  File "/srv/homeassistant/lib/python3.7/site-packages/homeassistant/helpers/entity.py", line 597, in async_request_call
    await coro
  File "/srv/homeassistant/lib/python3.7/site-packages/homeassistant/helpers/service.py", line 485, in _handle_entity_call
    await result
  File "/srv/homeassistant/lib/python3.7/site-packages/homeassistant/components/media_player/__init__.py", line 530, in async_turn_off
    await self.hass.async_add_job(self.turn_off)
  File "/usr/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/homeassistant/.homeassistant/custom_components/braviatv_psk/media_player.py", line 516, in turn_off
    self._braviarc.turn_off()
  File "/srv/homeassistant/lib/python3.7/site-packages/braviapsk/sony_bravia_psk.py", line 355, in turn_off
    self.send_req_ircc(self.get_command_code('PowerOff'))
  File "/srv/homeassistant/lib/python3.7/site-packages/braviapsk/sony_bravia_psk.py", line 282, in get_command_code
    self._refresh_commands()
  File "/srv/homeassistant/lib/python3.7/site-packages/braviapsk/sony_bravia_psk.py", line 273, in _refresh_commands
    if not resp.get('error'):
AttributeError: 'NoneType' object has no attribute 'get'
```

**Comment by gerard33** [comment]
Does it work when you turn of the TV with the power button on the Lovelace card?

And please share the automation you are using for this.

**Comment by wacki4** [comment]
I cannot turn it off - cause it is already off ;)

Here is the automation:
`- id: '1590434928904'
  alias: Opuszczenie domu
  description: ''
  trigger:
  - device_id: fc2850d648524e73baed22699c6c7767
    domain: binary_sensor
    entity_id: binary_sensor.czujnik_drzwi
    platform: device
    type: turned_off
  condition:
  - condition: device
    device_id: 656548ee624e4edba2868b823f9b5ae9
    domain: switch
    entity_id: switch.przedpokoj_dol
    type: is_on
  action:
  - scene: scene.zamknij_rolety
  - scene: scene.zgas_swiatla
  - device_id: 656548ee624e4edba2868b823f9b5ae9
    domain: switch
    entity_id: switch.przedpokoj_dol
    type: turn_off
  - data: {}
    entity_id: media_player.sonysalon
    service: media_player.turn_off
  - data: {}
    entity_id: media_player.sonysypialnia
    service: media_player.turn_off
  - data:
      option: Poza domem
    entity_id: input_select.stan_domu
    service: input_select.select_option`

**Comment by gerard33** [comment]
> I cannot turn it off - cause it is already off ;)

Not sure what you mean with this? That's not an answer to my question above is it?

And if the TV is on, can you then turn it off using `media_player.turn_off` in `/developer-tools/service` of Home Assistant?

**Comment by wacki4** [comment]
You are right, my mistake, misunderstood Your question ;)

Yes, when it is turn on - i can turn it off correctly. I can also correctly turn it on from hass.

**Comment by gerard33** [comment]
I checked and I get the same error when trying to turn of the TV using the `media_player.turn_off` command when the TV is already off.
I will make a new release which checks if the TV is off and in that case it wont send another off command. That prevents the error from happening.

In the meanhwile you can add a condition to your automation to only switch off the TV when it's actually on.

**Comment by gerard33** [comment]
Will be fixed in #40
